README.md

```diff
@@ -345,6 +344,17 @@ RP2040 chip according to how it is connected up on the Feather.
 [Waveshare RP2040 Zero]: https://www.waveshare.com/wiki/RP2040-Zero
 [waveshare-rp2040-zero]: https://github.com/rp-rs/rp-hal/tree/main/boards/waveshare-rp2040-zero
 
+### [waveshare-rp2040-lcd-0_96] - Board Support for the [Waveshare RP2040 LCD 0.96"]
+
+You should include this crate if you are writing code that you want to run on
+an [Waveshare RP2040 LCD 0.96"] - a very small RP2040 breakout board with USB-C, a 65K IPS LCD 160x80 and 1A battery charger from Waveshare.
+
+This crate includes the [rp2040-hal], but also configures each pin of the
+RP2040 chip according to how it is connected up on the Feather.
+
+[Waveshare RP2040 LCD 0.96"]: https://www.waveshare.com/wiki/RP2040-LCD-0.96
```

Comment:
Nitpick: The product name doesn't have a " in it

Reply:
You are right!
" removed